### PR TITLE
Update targets documentation for Frosting

### PR DESCRIPTION
**Pull request by pascalberger**
Update documentation how to run targets for Frosting

### CODE REVIEW

**Comment by nils-a** [comment]
@pascalberger from the text I inferr that there is currently no way in code to use another task as default than `Default`. Should we make that clearer?

**Comment by pascalberger** [comment]
> @pascalberger from the text I inferr that there is currently no way in code to use another task as default than `Default`. Should we make that clearer?

Yes, currently you can't define task in code. I tried to make it clear with the following sentence

> @pascalberger from the text I inferr that there is currently no way in code to use another task as default than `Default`. Should we make that clearer?

Yes this is currently not possible. I tried to make it clear with the `Cake Frosting by default will call a task named Default.` sentence. Now added an additional annotation. 

@nils-a Please let me know if you have better suggestions how it would be more clear for you.

File: input/docs/writing-builds/running-targets.md [hunk 1]
```diff
@@ -49,3 +52,31 @@ The following arguments will run the `Publish` target without running the `Build
 ```powershell
 --target=Publish --exclusive
 ```
+
+# Cake Frosting
+
+[Cake Frosting] by default will call a task named `Default`.
+
+:::{.alert .alert-info}
+In [Cake Frosting] it is not possible in to define the task which is run by default in your build script.
```
Comment (nils-a) [inline input/docs/writing-builds/running-targets.md:61]:
```suggestion
In [Cake Frosting] it is not possible in to define a task other than `Default`, which is run by default in your build script.
```

**Review by nils-a — APPROVED** [review]
One minor change, but not crucial. 
LGTM

**Pull request merged**